### DISCUSSION TITLE
Vontar X3 didn't start arbian

### DISCUSSION THREAD

**Discussion by howard180** [ophub/amlogic-s9xxx-armbian#2599] [Amlogic]
Hello! 
 
I try to start Armbian_24.11.0_amlogic_s905x3_bookworm_6.6.53_server_2024.10.02.img.gz  in Vontar X3 (4 gb , 32 Gb). But it doesn't start.  what i did before:
1. Set meson-sm1-hk1box-vontar-x3.dtb in FTD (file uEnv.txt)
2. Rename file from u-boot-s905x3.bin to u-boot.ext

**Comment by ophub** [comment]
No modification is required, you can directly use the s905x3 firmware to start

**Reply by howard180** [reply]
What you mean? What step i must to do?

**Comment by petesimon** [comment]
[howard180](https://github.com/howard180) try both SD card to boot and USB 3.0 port and USB 2.0 to boot. (3 attempts)

**Reply by howard180** [reply]
Tried all these cases, didn't help

**Reply by petesimon** [reply]
be more specific please in explaining what steps you have done. Did you use a toothpick and press/hold the reset button for 15 seconds?

also the currently installed firmware may be blocking the booting of 3rd party operating systems (armbian, emuelec, etc). or the bootloader is encrypted which also blocks some 3rd party things. This is true in my H96 Max X3 (S905x3) tv box. I had to downgrade the Android firmware to older firmware of the year 2022. Search online for an older stock Android firmware file. Another option is to install another Android firmware such as Aidan's-ROM, ( [link here](https://androidfilehost.com/?fid=2981970449027575291) ) overwriting the stock Android firmware, and this may allow booting Armbian. Below are two methods that I use to install any compatible Android firmware onto two tv boxes.

[T95 Max Plus / Burn Card Maker 2.03](https://forum.armbian.com/topic/46910-unbrick-amlogic-s905x3-t95-max-plus-tv-box-after-install-armbian-tutorial/)

and [H96 Max X3 / Boot Card Maker 1 with TWRP recovery](https://forum.armbian.com/topic/32488-h96-max-x3-armbian-install/#findComment-205379)

**Reply by petesimon** [reply]
>Search online for an older stock Android firmware file.

if this stock android firmware is older, then maybe try to downgrade your tv box by using this file.
[link here](https://disk.yandex.ru/d/RXwz1zpVCXgHGQ/%D0%9E%D0%B1%D1%80%D0%B0%D0%B7%20%D0%BF%D1%80%D0%BE%D1%88%D0%B8%D0%B2%D0%BA%D0%B8) 
https://disk.yandex.ru/d/RXwz1zpVCXgHGQ/

I also found [this information in freaktab](https://forum.freaktab.com/node/812507) forums and a mega link to another android firmware file. https://forum.freaktab.com/node/812507

*edit: and [another, year 2020](https://www.filefactory.com/file/dbeh0pfku0v/u218.VONTAR_X3_1000M_9.0.2020.04.06.10.48.img.7z), *edit: [another year 2020](https://mega.nz/file/OMdwgQZT#2yqcWMSVhnNzZ5qM0GVgtlXfWPFmSrNrmGtyW3Nojig)
